build failed with error `package javax.annotation does not exist`

```
Welcome to Gradle 7.4.1!

Here are the highlights of this release:
 - Aggregated test and JaCoCo reports
 - Marking additional test source directories as tests in IntelliJ
 - Support for Adoptium JDKs in Java toolchains

For more details see https://docs.gradle.org/7.4.1/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

> Task :clickhouse-core:compileJava
/data/home/caofengyu/spark-clickhouse-connector/clickhouse-core/build/generated/source/proto/main/grpc/xenon/protocol/grpc/ClickHouseGrpc.java:7: error: package javax.annotation does not exist
@javax.annotation.Generated(
                 ^
1 error

> Task :clickhouse-core:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':clickhouse-core:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 12s
9 actionable tasks: 5 executed, 4 up-to-date
```

openjdk 11.0.13_p8

--- a/build.gradle
+++ b/build.gradle
@@ -181,6 +181,8 @@ project(':clickhouse-core') {
             exclude group: "org.codehaus.mojo", module: "animal-sniffer-annotations"
         }
 
+        compileOnly "jakarta.annotation:jakarta.annotation-api:$jakarta_annotation_api_version"
+
         testFixturesApi "org.slf4j:slf4j-api:$slf4j_version"
         testFixturesApi "org.scalatest:scalatest_$scala_binary_version:$scalatest_version"
         testFixturesRuntimeOnly "com.vladsch.flexmark:flexmark-all:$flexmark_version"

--- a/gradle.properties
+++ b/gradle.properties
@@ -34,6 +34,10 @@ grpc_version=1.43.2
 guava_version=30.1.1-jre
 protobuf_version=3.19.2
 
+# javax annotations removed in jdk 11
+# fix build error with jakarta annotations
+jakarta_annotation_api_version=1.3.5
+
 # Test only
 clickhouse_jdbc_version=0.3.2-patch5
 testcontainers_scala_version=0.40.2

